skype/skype-ps/skype/Get-CsOnlineVoiceUser.md

```diff
@@ -55,11 +55,11 @@ The following table lists the output fields from `Get-CsOnlineVoiceUser` and the
 | Number | LineUri in the output from `Get-CsOnlineUser`. You can get same phone number format by doing LineUri.Replace('tel:+','')  |
 | Location | Use LocationId in the output from `Get-CsPhoneNumberAssignment -AssignedPstnTargetId <Identity>` as the input to `Get-CsOnlineLisLocation -LocationId` |
 
-In Teams PowerShell Module version 3.0 and later, the following improvements have been introduced for organizations using Teams: 
+In Teams PowerShell Module version 3.0 and later in commercial cloud (and Teams PowerShell Module versions 5.1.0 and later in GCCH and DOD), the following improvements have been introduced for organizations using Teams: 
```

Comment:
should be 5.0.1 for GCCH/DOD

Reply:
@pbafna03 it's 5.1.0 for voice cmdlet right? I said the same in MC post since it would be too short notice for 15th March

Reply:
nope this was enabled with 5.0.1, ref - https://learn.microsoft.com/en-us/microsoftteams/teams-powershell-release-notes

Reply:
updated here and MC post